flagging_site/blueprints/flagging.py

```diff
@@ -137,41 +137,7 @@ def output_model() -> str:
     return render_template('output_model.html', tables=reach_html_tables)
 
 
-class ReachApi(Resource):
+@bp.route('/api/v1/api', methods=['GET'])
```

Comment:
- Changed route to /api (flagging.py)

- Created new blueprint (api.py)

- Imported (blueprints/__init__.py)

- Added to ProductionConfig Blueprint list (config.py -- might want to double check syntax on that)

This now works as it was intended **API restored to former glory.** 

> Because the prefix takes care of the `/api` part of the route, the final route inside the decorator for rendering this page should just simply be `/`
- Am a little fuzzy on this part... which route are you referring to? I attempted this in flagging.py:
@bp.route('/', methods=['GET']) instead of: @bp.route('/api', methods=['GET'])
No dice.

Reply:
The reason that doesn't work in flagging.py is because you're routing to the blueprint with base `/`.

`/` + `/` = `/`

However, `/api` + `/` = `/api`.